src/overlays/actors/ovl_En_Syateki_Crow/z_en_syateki_crow.c

```diff
@@ -206,21 +206,21 @@ void EnSyatekiCrow_Fly(EnSyatekiCrow* this, PlayState* play) {
         this->currentPointIndex++;
     } else {
         this->isActive = false;
-        syatekiMan->guayFlags &= ~(1 << EN_SYATEKI_CROW_GET_INDEX(&this->actor));
+        syatekiMan->guayFlags &= ~(1 << SG_GUAY_GET_INDEX(&this->actor));
         EnSyatekiCrow_SetupWaitForSignal(this);
     }
 
     SkelAnime_Update(&this->skelAnime);
     this->actor.shape.yOffset = (fabsf(this->skelAnime.curFrame - 3.0f) * 150.0f) + 1700.0f;
-    if ((syatekiMan->perGameVar1.guaySpawnTimer % 90) == 0) {
+    if ((syatekiMan->guayAppearTimer % 90) == 0) {
```

Comment:
Outside the scope of this PR and definitely don't change here, but just writing down that I sort of wish we had something more to indicate that a variable is a union/shared between others. Even a convention like `syatekiMan->u.guayAppearTimer` idk.

Reply:
Whether unions should be named or unnamed, or whether they were even *allowed* to be unnamed at all, seemed to change wildly depending on whatever the whims of any given lead were at any given point. I would absolutely *love* some degree of consistency or guidance here, because I've been told to both add a name to a union and remove it at different points in time. In particular, I always hated the `perGameVarX` thing here, and I only did it to get it merged; I removed it here because I saw that you in particular were using unnamed unions in EnTest6, and that this might be my chance to get rid of something I always disliked.

Reply:
Yeah, that's fair. It's definitely not consistent and people have different preferences. OoT also tends to avoid unions much more than MM for reused variables, so it'll be an interesting discussion moving forward.

Reply:
A couple of actors in OoT do a thing where they use unions in all but name. For example, `z_en_tite` has struct variables with names like `actionVar1`, then those are redefined to be more specific names like so:
```c
// EnTite_Attack (vQueuedJumps also used by EnTite_MoveTowardPlayer)
#define vAttackState actionVar1
#define vQueuedJumps actionVar2

//  EnTite_FlipOnBack
#define vOnBackTimer actionVar1
#define vLegTwitchTimer actionVar2
```

I actually started with this approach with ObjTokeidai (an actor with a huge number of reused variables), and I was told to use unions instead. So that's what I've been doing with all my actors, and in doing so, I've repeatedly run into mixed messaging about *how* I should be using unions.

Like you said, this isn't really in scope for this PR. I would love to see and participate in discussions about how we should be handling these sort of reused variables. But I would *also* love to get rid of this thing I never truly liked in the first place, hence why I snuck it into this PR. I can revert it if desired, though as you might guess, I would really prefer it if I didn't.

Reply:
I'm fine with leaving it to personal preference until a consensus is reached.

Tbh that's the first time I've seen that trick used in OoT. I knew the defines with the `v` prefix was used for locally scoped temps, but not for struct members (other than overlay effects which uses that trick for `this->regs[X]`). Not sure what that offers over unions except for dispersing the information to more places.

I used to like the `v` prefix defines for multi-use temps but am not as fond of it anymore.